<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfCheckClass</class>
 <widget class="QMainWindow" name="PerfCheckClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1615</width>
    <height>1072</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PerfCheck</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTableView" name="fileTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="ChartView" name="chart">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QTableView" name="dataTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="statsLayout"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="printButton">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1615</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QFrame</extends>
   <header>ChartView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PerfCheck.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>printButton</sender>
   <signal>clicked()</signal>
   <receiver>PerfCheckClass</receiver>
   <slot>print_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>PerfCheckClass</receiver>
   <slot>datasetClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>931</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>1045</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>print_pressed()</slot>
  <slot>datasetClicked(QModelIndex)</slot>
 </slots>
</ui>
